Bottom: 6869788182028a94bb620f43520b4eb342a4f7bf
Top:    0d74cf69f43f01fca87746c91f2e2e654e65f353
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 18:19:08 +0000

Refresh of attempt-to-refactor-the-step

---

diff --git a/src/hid/step/object3d_step.c b/src/hid/step/object3d_step.c
index ec01561..3464ff2 100644
--- a/src/hid/step/object3d_step.c
+++ b/src/hid/step/object3d_step.c
@@ -36,7 +36,7 @@ presentation_style_assignments_from_appearance (step_file *step, appearance *app
 }
 
 void
-object3d_export_to_step (object3d *object, char *filename)
+object3d_export_to_step (object3d *object, const char *filename)
 {
   FILE *f;
   step_file *step;
diff --git a/src/hid/step/object3d_step.h b/src/hid/step/object3d_step.h
index 049fc6b..1d2d5cd0 100644
--- a/src/hid/step/object3d_step.h
+++ b/src/hid/step/object3d_step.h
@@ -1 +1 @@
-void object3d_export_to_step (object3d *object, char *filename);
+void object3d_export_to_step (object3d *object, const char *filename);
